src/binding.cc

```diff
@@ -315,6 +316,11 @@ Napi::Value BpfObjGet(const CallbackInfo& info) {
     return ToStatus(env, bpf_obj_get(path.c_str()));
 }
 
+Napi::Value GetNumPossibleCPUs(const CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    return ToStatus(env, libbpf_num_possible_cpus());
```

Comment:
nit: we can't use ToStatus here, the function return the error code directly (not through errno)